src/Recruit.Vacancies.Client/Infrastructure/QueryStore/QueryStore.cs

```diff
@@ -1,32 +1,46 @@
-using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Esfa.Recruit.Vacancies.Client.Domain.Projections;
 using Esfa.Recruit.Vacancies.Client.Domain.QueryStore;
 using Esfa.Recruit.Vacancies.Client.Infrastructure.Mongo;
 using Microsoft.Extensions.Options;
 using MongoDB.Driver;
-using System.Collections.Generic;
 using System.Threading.Tasks;
 
 namespace Esfa.Recruit.Vacancies.Client.Infrastructure.QueryStore
 {
     internal sealed class QueryStore : MongoDbCollectionBase, IQueryStoreReader, IQueryStoreWriter
     {
         private const string Database = "recruit";
-        private const string Collection = "vacancies";
+        private const string Collection = "queryViews";
 
         public QueryStore(IOptions<MongoDbConnectionDetails> details)
             : base(Database, Collection, details)
         {
         }
 
-        public async Task<IEnumerable<Vacancy>> GetVacanciesAsync(string employerAccountId)
+        public async Task<Dashboard> GetDashboardAsync(string key)
         {
-            var filter = Builders<Vacancy>.Filter.Eq(v => v.EmployerAccountId, employerAccountId);
-            filter = filter & Builders<Vacancy>.Filter.Eq(v => v.IsDeleted, false);
+            var filter = Builders<Dashboard>.Filter.Eq(d => d.ViewKey, key);
```

Comment:
As its mongo shouldn't we be targeting the 'id' field?

Reply:
Yeah I had issues with this as Mongo requires the Id to be untouched but at the same time it doesn't make identifying the item in Mongo easier.

Reply:
If we make the id the EmployerAccountId its not going to change so will remain untouched